templates/cf.yml

```diff
@@ -97,6 +97,10 @@ jobs:
     properties:
       metron_agent:
         zone: z1
+      consul:
+        agent:
+          services:
+            etcd: {name: cf-etcd}
```

Comment:
Just wanted to point this out, we named this `cf-etcd` in comparison to `other-etcd`. The certs should have common name of `cf-etcd.service.cf.internal`